src/js/components/Box/doc.js

```diff
@@ -21,13 +20,14 @@ export default Box => schema(Box, {
       `How to align along the cross axis when contained in a Box or along
       the column axis when contained in a Grid.`,
     ],
-    // appCentered - removed
     background: [
       PropTypes.oneOfType([
         PropTypes.string,
         PropTypes.shape({
+          color: PropTypes.string,
           dark: PropTypes.bool,
```

Comment:
Might want to clarify that `dark` isn't needed when using `color`.

Reply:
I will add that in the docs.